packages/tables/src/Table.php

```diff
@@ -373,6 +382,17 @@ public function getRecordAction(): ?string
         return $this->recordAction;
     }
 
+    public function getRecordClasses(Model $record): null|string|array
+    {
+        $callback = $this->getRecordClassesUsing;
+
+        if (! $callback) {
+            return null;
+        }
+
+        return $callback($record);
```

Comment:
Let's wrap the callback return value and always return `?array` (`Arr::wrap()`).

Reply:
`array_merge` doesn't accept `null`, so I made the return type `array`

Reply:
Even better!